### ISSUE TITLE
DNSCrypt new location in github

### ISSUE THREAD

**Issue by zznosar** [alterstep/dnscrypt-osxclient#161]
Hi All,

Will there be an update to the client to point to the new DNS resolvers location in github?

The project is now maintained here:

https://github.com/dyne/dnscrypt-proxy

*****************************************************************************************************************

Before creating an issue, please read the FAQ: https://github.com/alterstep/dnscrypt-osxclient/blob/master/FAQ.md

Informations and debug logs as explained here are almost always more useful that screenshots!

Thank you!

*****************************************************************************************************************

**Comment by tiferrei** [comment]
Any updates on this?

Thanks,
tiferrei

**Comment by JayBrown** [comment]
See #165 with links to two BitBar plugins that you can use to manage the new dnscrypt-proxy version 2.

As for a dedicated GUI client software, there hasn't been any news.

**Comment by Jolg42** [comment]
I think you made a mistake @zznosar the repository for dnscrypt-proxy is https://github.com/jedisct1/dnscrypt-proxy

**Comment by JayBrown** [comment]
You're right. The link in the OP is to version 1 of dnscrypt-proxy, which is just mirrored for posterity.
